images/scripts/start-kafka.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+export KAFKA_CONFIGURATION=$(cat /config/server.properties)
```

Comment:
not sure about that, can't we just use file contents inside `images/scripts/generate-configuration.sh` instead of passing them via env var? `KAFKA_CONFIGURATION` doesn't seem to be used anywhere else, or?

Reply:
yeah now it's not used anywhere else, moved it inside